internal/collector/vector/visitors.go

```diff
@@ -8,9 +8,9 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
-func CollectorVisitor(collectorContainer *corev1.Container, podSpec *corev1.PodSpec, resNames *factory.ForwarderResourceNames, namespace string) {
+func CollectorVisitor(collectorContainer *corev1.Container, podSpec *corev1.PodSpec, resNames *factory.ForwarderResourceNames, namespace, logLevel string) {
 	collectorContainer.Env = append(collectorContainer.Env,
-		corev1.EnvVar{Name: "VECTOR_LOG", Value: "WARN"},
```

Comment:
the change from warn to default will bump the log output of the collector making it more verbose.  I wonder if that is an issue?

Reply:
I did that because vector's docs state the default is info. I can make the default to warn instead so it will keep the same behavior as we previously had.